custom-login/src/App.vue

```diff
@@ -69,13 +69,21 @@
     >
       <router-view/>
     </div>
+    <AuthRequiredModal ref="authRequiredModal" />
   </div>
 </template>
 
 <script>
+import AuthRequiredModal from './components/AuthRequiredModal.vue';
 export default {
   name: 'app',
+  components: {
+    AuthRequiredModal,
+  },
   methods: {
+    onAuthRequired(oktaAuth) {
+      this.$refs.authRequiredModal.onAuthRequired(oktaAuth);
```

Comment:
the `authRequiredModal ` should not own the logic for `App initialization stage`. 

try with
```
if (!oktaAuth.authStateManager.getPreviousAuthState()?.isAuthenticated) {
        // App initialization stage
        router.push('/login');
      } else {
        // Ask the user to trigger the login process during token autoRenew process
        this.$refs.authRequiredModal.show()
      }
```

Reply:
Done